OpenRA.Game/Settings.cs

```diff
@@ -77,7 +77,8 @@ public class ServerSettings
 		[Desc("Enable client-side report generation to help debug desync errors.")]
 		public bool EnableSyncReports = false;
 
-		public string TimestampFormat = "s";
+		[Desc("Sets the timestamp format.")]
+		public string TimestampFormat = "yyyy-MM-ddTHH:mm:ssZ"; // https://xkcd.com/1179/
```

Comment:
:smile: 
But please remove the comment.

Reply:
What if the comment was `ISO 8601` instead?

Reply:
Done.